### DISCUSSION TITLE
Request feature to be add  : set a color to the vscode windows per project.

### DISCUSSION THREAD

**Discussion by davidfrazao** [alefragnani/vscode-project-manager#542] [Ideas]
hi 
Could it be possible to have a feature that able the user to choose the colour theme of the project, this way we could have different colour themes for different projects.

![imagem](https://user-images.githubusercontent.com/1057020/131156996-af81d49f-66c6-48bc-ac02-7e79c70ea4a2.png)

thx in advance

**Comment by alefragnani** [comment]
Hi @davidfrazao ,

Thank you for your suggestion, but I’m trying to avoid mixing the management (catalog) of projects, with its settings. This would end up creating another level of complexity on how the settings are applied when you open a workspace.

`<Project Manager Settings><Workspace Settings><Folder Settings><User Settings>`

I’m not saying this won’t happen in the future, because in fact I think it wouldn’t be that hard to achieve, but unless something changes priorities, it’s not in the roadmap for upcoming releases.

In the meantime, do you know the [Peacock](https://marketplace.visualstudio.com/items?itemName=johnpapa.vscode-peacock) extension? I guess it could fit perfectly in your scenario.

Hope this helps

**Reply by 0xEzis** [reply]
Hi @alefragnani, would it be possible to add colored dots, similar to those on macOS, to associate projects with specific colors?

For example, I use a primary color for each of my projects, and it would be great to see them directly in the project manager tab. 😉

**Comment by davidfrazao** [comment]
hi @alefragnani 

Thx for the feedback, about the Peacock extension, yes I know and I am using it at the moment.

thx ... again 

best regard
